parser/gen/golang/errors.go

```diff
@@ -72,4 +73,18 @@ func (E *Error) String() string {
 	}
 	return w.String()
 }
+
+func (e *Error) Error() string {
+	w := new(bytes.Buffer)
+	fmt.Fprintf(w, "Error in S%d: %s, %s", e.StackTop, token.TokMap.TokenString(e.ErrorToken), e.ErrorToken.Pos.String())
```

Comment:
Seems like the stacktop is still printed out here?
How does it get eliminated in the final output?